### PR TITLE
Feature/0002 approval

### PR DESCRIPTION
**Pull request by BrianMRO**
Added standard Attributes to Work Order Class and Work Order screen.

Enhanced Work Order screen:
   - Added attributes
   - Adjusted U/M to pull from the InventoryItem record BaseUnit field on Materials tab
   - Adjusted screen structure in accordance with Manufacturing BOM layout (Known issue to resolve at a later time -> child tabs of the Operations tab are not updating when selecting a different operation.  Compared extensively to Manufacturing BOM screen and unable to identify root cause.  Tabs update when on that tab, but not when on another tab. - Documented as Issue #23.)

Enhanced workflows for Work Order screen.

### CODE REVIEW

File: CMMS/Graph/Entry/WOOrderEntry.cs [hunk 1]
```diff
@@ -18,25 +22,29 @@ public class WOOrderEntry : PXGraph<WOOrderEntry, WOOrder>
             .View CurrentDocument;
 
         [PXViewName(Messages.ViewTransactions)]
+        [PXImport(typeof(WOLine))]
```
Comment (lekker-solutions) [inline CMMS/Graph/Entry/WOOrderEntry.cs:25]: I might be totally mistaken, but dont these need to be PXImport(typeof(MainDAC)) ? so [PXImport(typeof(WOOrder))]

File: CMMS/Graph/Entry/WOOrderEntry.cs [hunk 2]
```diff
@@ -134,6 +158,39 @@ protected virtual IEnumerable Schedule(PXAdapter adapter)
...
+        protected virtual IEnumerable Complete(PXAdapter adapter) => adapter.Get<WOOrder>();
+        #endregion
+
+        #endregion
+
+        #region Events
+        #region WOOrder_RequestApproval_FieldDefaulting
+        protected void _(Events.FieldDefaulting<WOOrder.requestApproval> e)
+        {
+            e.NewValue = Setup.Current.WORequestApproval;
```
Comment (lekker-solutions) [inline CMMS/Graph/Entry/WOOrderEntry.cs:181]: Can we just make this an attribute on WOOrder.requestApproval?

[PXDefault(typeof(WOSetup.woRequestApproval))]

Comment (jvanhoesen) [reply CMMS/Graph/Entry/WOOrderEntry.cs]: Resolved in latest commit

File: CMMS/Graph/Entry/WOOrderEntry.cs (hunk 1 above)
Comment (jvanhoesen) [reply CMMS/Graph/Entry/WOOrderEntry.cs]: Resolved in latest commit

**Review by jvanhoesen — APPROVED** [review]

**Pull request merged**